Running make generates Makefile warning and bazel debug messages

Running 'make build' produces a warning:
```sh
Makefile:61: warning: overriding recipe for target 'clean'
Makefile:47: warning: ignoring old recipe for target 'clean'
```
There are also a number of bazel debug messages for renaming 'files' attribute as 'srcs' for some targets. For example:
```
DEBUG: /home/elevran/.cache/bazel/_bazel_elevran/671652e334809f32387d36d0b89e2b19/external/bazel_tools/tools/build_defs/pkg/pkg.bzl:197:9: @//pilot/tools/deb:agent-bin: you provided a non dictionary to the pkg_tar `files` attribute. This attribute was renamed to `srcs`. Consider renaming it in your BUILD file.
...
```

--- a/Makefile
+++ b/Makefile
@@ -57,10 +57,6 @@ docker:
 push: checkvars
 	@$(TOP)/bin/push $(HUB) $(TAG)
 
-clean:
-	@bazel clean
-
-
 artifacts: docker
 	@echo 'To be added'
 

--- a/Makefile
+++ b/Makefile
@@ -30,6 +30,8 @@ ifneq ($(strip $(TAG)),)
 	tag =-tag ${TAG}
 endif
 
+.DEFAULT_GOAL := build
+
 checkvars:
 	@if test -z "$(TAG)"; then echo "TAG missing"; exit 1; fi
 	@if test -z "$(HUB)"; then echo "HUB missing"; exit 1; fi

--- a/broker/docker/BUILD
+++ b/broker/docker/BUILD
@@ -6,7 +6,7 @@ load("//broker/docker:broker_docker.bzl", "broker_docker_build")
 pkg_tar(
     name = "broker_tar",
     extension = "tar.gz",
-    files = [
+    srcs = [
         "//broker/cmd/brks",
     ],
     mode = "0755",

--- a/docker/BUILD
+++ b/docker/BUILD
@@ -6,7 +6,7 @@ load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
 pkg_tar(
     name = "hop_tar",
     extension = "tar.gz",
-    files = [
+    srcs = [
         "//tests/e2e/apps/hop/hop-server",
     ],
     mode = "0755",

--- a/mixer/docker/BUILD
+++ b/mixer/docker/BUILD
@@ -10,7 +10,7 @@ load("//mixer/docker:cacerts.bzl", "cacerts")
 pkg_tar(
     name = "mixer_tar",
     extension = "tar.gz",
-    files = [
+    srcs = [
         "//mixer/cmd/server:mixs",
     ],
     mode = "0755",

--- a/mixer/example/servicegraph/docker/BUILD
+++ b/mixer/example/servicegraph/docker/BUILD
@@ -6,7 +6,7 @@ load("//mixer/docker:mixer_docker.bzl", "mixer_docker_build")
 pkg_tar(
     name = "mixer_servicegraph_tar",
     extension = "tar.gz",
-    files = [
+    srcs = [
         "//mixer/example/servicegraph/cmd/server",
     ],
     mode = "0755",

--- a/mixer/example/servicegraph/js/BUILD
+++ b/mixer/example/servicegraph/js/BUILD
@@ -5,7 +5,7 @@ load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 pkg_tar(
     name = "js_tar",
     extension = "tar.gz",
-    files = glob(
+    srcs = glob(
         ["**"],
         exclude = ["BUILD"],
     ),

--- a/mixer/testdata/BUILD
+++ b/mixer/testdata/BUILD
@@ -5,7 +5,7 @@ load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 pkg_tar(
     name = "configs_tar",
     extension = "tar.gz",
-    files = glob([
+    srcs = glob([
         "**/*.yml",
     ]),
     mode = "0755",

--- a/pilot/tools/deb/BUILD
+++ b/pilot/tools/deb/BUILD
@@ -2,7 +2,7 @@ load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
 
 pkg_tar(
     name = "agent-bin",
-    files = [
+    srcs = [
         "//pilot/cmd/pilot-agent",
     ],
     mode = "0755",

